### DISCUSSION TITLE
Prevent abuse of namespace in crates.io

### DISCUSSION THREAD

**Discussion by ulvido** [rust-lang/crates.io#4843] [General]
for example this [user](https://crates.io/users/fall-water) uploads 70 crates in 2 days, all crates have one word names which are empty and useless crates.

we need to prevent that toxic behaviour.

thank you.

**Comment by carols10cents** [comment]
This user's accounts have been locked because they've published these in violation of our policies. We are working to remove the crates soon.

Any change to our current policies needs someone to write and manage an RFC. https://github.com/rust-lang/rfcs
